sample/kotlinapp/src/main/java/com/marcinmoskala/kotlinapp/notification/NotificationPublisher.kt

```diff
@@ -16,18 +19,36 @@ class NotificationPublisher : BroadcastReceiver() {
 
     override fun onReceive(context: Context, intent: Intent) {
         ActivityStarter.fill(this, intent)
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
+        {
```

Comment:
It doesn't look like Kotlin style
https://kotlinlang.org/docs/reference/coding-conventions.html